Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerKnownBuildIssues.ps1

```diff
@@ -0,0 +1,125 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+. $PSScriptRoot\Add-AnalyzedResultInformation.ps1
+. $PSScriptRoot\..\Helpers\Invoke-CatchActions.ps1
+
+Function Invoke-AnalyzerKnownBuildIssues {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory = $true)]
```

Comment:
Let's add `[ValidateNotNullOrEmpty()]` to make sure that no `$null` or empty values are passed

Reply:
A Mandatory parameter already can't be null or empty.

![image](https://user-images.githubusercontent.com/4518572/146984213-f5f5cd4c-d272-4487-a982-55e4c37e339b.png)

Reply:
Can you pass “ “ (with blank space) instead of “”? Does that work for required parameters?

Reply:
Could probably pass an empty value since it is string param. But what we are passing as a `ref` not sure how that is possible to be a problem.

Reply:
Yes you can pass a space. That satisfies the requirement of Mandatory. ValidateNotNullOrEmpty doesn't block that, either, because it's not null or empty, it's a space.

You can't pass an empty string, even when it's a string type (see above).

Reply:
![image](https://user-images.githubusercontent.com/4518572/146989910-d3601478-55a5-4ffc-a68c-417bcffac58c.png)

Reply:
Got it. So let’s do it without the validation.